src/hotspot/os/windows/os_windows.cpp

```diff
@@ -4105,7 +4115,7 @@ static BOOL CALLBACK init_crit_sect_call(PINIT_ONCE, PVOID pcrit_sect, PVOID*) {
   return TRUE;
 }
 
-int os::win32::exit_process_or_thread(Ept what, int exit_code) {
+static void exit_process_or_thread(Ept what, int code) {
```

Comment:
exit_code was clearer and avoids changes below

Reply:
Noted, will return the name to exit_code